### PR TITLE
Doc Tech Edits

### PR DESCRIPTION
**Pull request by ScottEvil**
Reviewed the documentation, corrected grammar, spelling, and focused on readability. This addresses Issue #706 and Issue #709. This also fixes index hierarchy problems with the Ingest section and the ingeststats table formatting.

### CODE REVIEW

File: docs/content/015-theorydecomposition.adoc [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 It should be noted that sometimes more simplistic algorithms will not fully decompose – but would instead represent
-this as 70-121, or even 64->127 (the smallest “unit cell” this box fits in). As you can see this would result in
+this as 70->121, or even 64->127 (the smallest “unit cell” this box fits in). As you can see, this would result in
 scanning many extraneous cells.
 
-At some point – with high precision high dimensionality curves – the number of possible unit cells can become to large
-too deal with – in this case GeoWaves optimizes this by treating the curve as a “lower cardinality” curve than it
-actually is – so the unit cell size might not be 1, but instead 64, 128, 1024, etc. This allows the user to still
+At some point, with high precision high dimensionality curves, the number of possible unit cells can become to large
+too deal with. In such a case, GeoWaves optimizes this by treating the curve as a “lower cardinality” curve than it
```
Comment (rfecher) [inline docs/content/015-theorydecomposition.adoc]: too large to deal with

Comment (rfecher) [inline docs/content/015-theorydecomposition.adoc]: GeoWave optimizes this

File: docs/content/020-screenshots.adoc [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [[screenshots]]
 <<<
-== Screenshots
+== Example screenshots
 
-The screenshots below are of data loaded from the various attributed data sets into a Geowave instance, processed
+The screenshots below are of data loaded from various attributed data sets into a Geowave instance, processed
```
Comment (rfecher) [inline docs/content/020-screenshots.adoc]: GeoWave

File: docs/content/020-screenshots.adoc [hunk 3]
```diff
@@ -68,7 +68,7 @@ More information on this data set is available at: http://research.microsoft.com
 
 ==== T-drive at city scale
 
-Below are renderings of the t-drive data. They display both the raw points, as well as the results of a Geowave kernel
+Below are renderings of the t-drive data. They display the raw points along with the results of a Geowave kernel
```
Comment (rfecher) [inline docs/content/020-screenshots.adoc]: GeoWave

**Comment by ScottEvil** [comment]
@rfecher Made fixes according to your comments.

**Pull request merged**